docs/source/conf.py

```diff
@@ -35,6 +35,8 @@
     "sphinx.ext.todo",
     "sphinx.ext.viewcode",
     "nbsphinx",
+    "sphinx_copybutton",
+    "autoapi.extension",
```

Comment:
I cannot get rid of the warning either. It is because simulate and the other functions also exist under `sid.` and in their modules as well. Documenting the same function twice is bad.